web.py

```diff
@@ -4,17 +4,34 @@
 from github import Github
 from joiner_scripts.joiner import AqGpsJoiner
 import datetime
+import base64
 
 app = Flask(__name__)
 
 APP_ROOT = os.path.dirname(os.path.abspath(__file__))
 
+#Variables
+g = Github("fa0dfb5e6103c9bf91cbbcc22e3c1aa5708cbdb4")
+repo = g.get_user().get_repo('woaq')
+
 @app.route('/')
 def index():
-    return render_template("upload.html")
+    contents = repo.get_contents("_Posts")
+    #print(contents)
+    #csvFileTemp = repo.get_contents('/_Posts/'+filename)
+    #print('CSVFile below:')
+    # while len(contents) > 1:
+    #     file_content = contents.pop(0)
+    #     if file_content.path == "_Posts":
+    #         contents.extend(repo.get_contents(file_content.path))
+    #     else:
+    #         print(file_content)
+    return render_template("home.html",databaseFiles=contents)
 #Runs once file is uploaded
-@app.route("/upload", methods=['POST'])
+@app.route("/upload", methods=['POST','GET'])
 def upload():
+    if (request.method == 'GET'):
+        return render_template("upload.html")
     now = datetime.datetime.now()
```

Comment:
If there is a function which assumes that it is getting a post call, it should be wrapped in 'elif (request.method == 'POST')